com.ibm.streamsx.topology/opt/python/include/splpy.h

```diff
@@ -82,13 +82,13 @@ namespace streamsx {
 
 
     /**
-     * Convert a SPL blob into a Python Byte string 
+     * Convert a SPL blob into a Python Memory view object.
      */
     inline PyObject * pyAttributeToPyObject(const SPL::blob & attr) {
       long int sizeb = attr.getSize();
       const unsigned char * bytes = attr.getData();
 
-      return PyBytes_FromStringAndSize((const char *)bytes, sizeb);
+      return PyMemoryView_FromMemory((char *) bytes, sizeb, PyBUF_READ);
```

Comment:
so for 2.7 python the PyMemoryView_FromMemory() call needs to be 
PyObject* PyBuffer_FromMemory((void *)bytes, sizeb);   ?